style/js/shield_text.js

```diff
@@ -206,14 +206,42 @@ export function drawBannerText(ctx, text, bannerIndex) {
 
   ctx.textBaseline = "top";
   ctx.font = Gfx.shieldFont(textLayout.fontPx);
-  ctx.shadowColor = "white";
-  ctx.shadowBlur = 10;
+  ctx.shadowColor = "rgba(250, 246, 242, 1)";
+  ctx.shadowOffsetX = -2;
+  ctx.shadowOffsetY = -2;
 
   ctx.fillText(
     text,
     textLayout.xBaseline,
     textLayout.yBaseline + bannerIndex * ShieldDef.bannerSizeH
   );
+  
+  ctx.shadowOffsetX = -2;
+  ctx.shadowOffsetY = 2;
+  
+  ctx.fillText(
+    text,
+    textLayout.xBaseline,
+    textLayout.yBaseline + bannerIndex * ShieldDef.bannerSizeH
+  );
+  
+  ctx.shadowOffsetX = 2;
+  ctx.shadowOffsetY = -2;
+  
+  ctx.fillText(
+    text,
+    textLayout.xBaseline,
+    textLayout.yBaseline + bannerIndex * ShieldDef.bannerSizeH
+  );
+  
+  ctx.shadowOffsetX = 2;
+  ctx.shadowOffsetY = 2;
+  
+  ctx.fillText(
+    text,
+    textLayout.xBaseline,
+    textLayout.yBaseline + bannerIndex * ShieldDef.bannerSizeH
+  );
```

Comment:
Just a suggestion to replace the copy/paste with a loop:

```suggestion
  ctx.shadowColor = "rgba(250, 246, 242, 1)";
  [-2, 2].forEach((offsetX) => {
    [-2, 2].forEach((offsetY) => {
      ctx.shadowOffsetX = offsetX;
      ctx.shadowOffsetY = offsetY;

      ctx.fillText(
        text,
        textLayout.xBaseline,
        textLayout.yBaseline + bannerIndex * ShieldDef.bannerSizeH
      );
    });
  });
```